packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts

```diff
@@ -222,10 +235,17 @@ export async function __getBulkThreads(
       addressesReacted: t.addresses_reacted
         ? t.addresses_reacted.split(',')
         : [],
+      reactedProfileName: t.reacted_profile_name?.split(','),
+      reactedProfileAvatarUrl: t.reacted_profile_avatar_url?.split(','),
+      reactedAddressLastActive: t.reacted_address_last_active?.split(','),
       reactionType: t.reaction_type ? t.reaction_type.split(',') : [],
       marked_as_spam_at: t.marked_as_spam_at,
       archived_at: t.archived_at,
       latest_activity: t.latest_activity,
+      profile_id: t.profile_id,
+      avatar_url: t.avatar_url,
+      address_last_active: t.address_last_active,
+      profile_name: t.profile_name,
```

Comment:
not my area but maybe it would be good to close it in an object for better encapsulation and readability, similarly to `Address` above.